content/guides/pedestal-with-datomic.adoc

```diff
@@ -0,0 +1,170 @@
+= Using Pedestal With Datomic
+Mateus Carvalho
+2022-06-14
+:jbake-type: page
+:toc: macro
+:icons: font
+:section: reference
+
+ifdef::env-github,env-browser[:outfilessuffix: .adoc]
+
+toc::[]
+_by {author}_, _{revdate}_
+
+== What You Will Learn
+After reading this guide you will be able to:
+
+- Create a Pedestal service connecting to Datomic.
+- Test your service using Pedestal's test helpers.
+
+== Guide Assumptions
+This tutorial extends the link:hello-world[Hello World Service] to use strings retrieved
+from Datomic. We will start from the "hello" service that we created
+in our first tutorial.
+
+This tutorial assumes that you have Datomic downloaded already.
+If not, hop over to the https://www.datomic.com/get-datomic.html[download site].
+
+== Add Datomic to the Project
+After downloading the datomic
```

Comment:
this looks truncated, should be something like "After downloading and installing the Datomic distibution ..."